lib/oauth2/mac_token.rb

```diff
@@ -29,6 +29,9 @@ def self.from_access_token(token, secret, options = {})
     # @option opts [FixNum, String] :algorithm (hmac-sha-256) the algorithm to use for the HMAC digest (one of 'hmac-sha-256', 'hmac-sha-1')
     def initialize(client, token, secret, opts = {})
       @secret = secret
+      @seq_nr = SecureRandom.random_number(2 ** 64)
```

Comment:
> Sequence numbers fall in the range 0 through 2^64 - 1 and wrap to zero following the value 2^64 - 1.

As [SecureRandom.random_number(n)](https://apidock.com/ruby/SecureRandom/random_number/class) generate number => 0 and < n, we needn'nt minus 1